src/js/media-container.js

```diff
@@ -66,21 +62,10 @@ template.innerHTML = `
     }
 
     /* Video specific styles */
-    :host(:not([audio])) {
-      aspect-ratio: var(--media-aspect-ratio, auto 3 / 2);
```

Comment:
So by removing this, we're saying "just use the `aspect-ratio` CSS, don't bother with a css var", right?

Reply:
yup